### PR TITLE
H60 updates

### PR DESCRIPTION
**Pull request by hineybush**
## Description

Slight update to the backlight config, wrapped in changes from #16811 as well, closed that PR.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/hineybush/h60/config.h [hunk 1]
```diff
@@ -38,21 +38,28 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define DIODE_DIRECTION COL2ROW
 
 #define BACKLIGHT_PIN B7
-#define BACKLIGHT_BREATHING
-#define BACKLIGHT_LEVELS 3
+#define BACKLIGHT_LEVELS 12
+#define BACKLIGHT_ON_STATE 1
```
Comment (fauxpark) [inline keyboards/hineybush/h60/config.h:42]:
```suggestion
```

File: keyboards/hineybush/h60/keymaps/via/keymap.c [hunk 2]
```diff
@@ -3,35 +3,27 @@
...
-    _FN1,
-    _FN2,
-    _FN3
-};
-
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
-    [_BASE] = LAYOUT_all(
-    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,    KC_EQL,     KC_BSPC,  KC_TRNS,
+    [0] = LAYOUT_all(
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/via/keymap.c:8]: Technically, you can do this too:
```suggestion
    LAYOUT_all(
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/hineybush/h60/keymaps/via/keymap.c [hunk 3]
```diff
@@ -3,35 +3,27 @@
...
-    [_BASE] = LAYOUT_all(
-    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,    KC_EQL,     KC_BSPC,  KC_TRNS,
+    LAYOUT_all(
+    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,    KC_EQL,     KC_TRNS,  KC_BSPC,
     KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,     KC_LBRC,    KC_RBRC,    KC_BSLS,
     KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN,  KC_QUOT,    KC_ENT,
     KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT,    KC_TRNS,
     KC_LCTL, KC_LGUI, KC_LALT, KC_TRNS,                   KC_SPC,           KC_TRNS, KC_RALT, MO(1),    KC_RGUI,    KC_RCTL),
-    [_FN1] = LAYOUT_all(
+    [1] = LAYOUT_all(
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/via/keymap.c:14]:
```suggestion
    LAYOUT_all(
```

File: keyboards/hineybush/h60/keymaps/via/keymap.c [hunk 4]
```diff
@@ -3,35 +3,27 @@
...
     KC_TRNS, BL_TOGG, BL_DEC,  BL_INC,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RESET,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_MPLY,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
-    [_FN2] = LAYOUT_all(
+    [2] = LAYOUT_all(
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/via/keymap.c:20]:
```suggestion
    LAYOUT_all(
```

File: keyboards/hineybush/h60/keymaps/via/keymap.c [hunk 5]
```diff
@@ -3,35 +3,27 @@
...
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
-    [_FN3] = LAYOUT_all(
+    [3] = LAYOUT_all(
```
Comment (fauxpark) [inline keyboards/hineybush/h60/keymaps/via/keymap.c:26]:
```suggestion
    LAYOUT_all(
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**